skl2onnx/validate.py

```diff
@@ -0,0 +1,827 @@
+# -------------------------------------------------------------------------
```

Comment:
Could you increase test coverage on this file? Currently 74 lines in this file haven't been tested by unit tests.
skl2onnx/validate.py                                         399     74    81%